control/autopilot/include/autopilot/autopilot.h

```diff
@@ -137,10 +138,11 @@ class AutoPilot
 
   state_predictor::StatePredictor state_predictor_;
 
-  position_controller::PositionController base_controller_;
-  position_controller::PositionControllerParams base_controller_params_;
+  Tcontroller base_controller_;
```

Comment:
I named this base controller because I thought that there might be new controllers that are only used for certain tasks but some of the basic tasks are always executed with the base controller (like start land etc.). @foehnph would something like this make sense (and then have two controller instances) or not?

Reply:
I agree with @mfaessle, it makes it easier for you @foehnph since you don't need to implement the velocity mode in mpc, no?

Reply:
not really... then you would have to switch between controllers, implement state machine for that and generally reimplement a shitload of things if you want to use the autopilot with the new controller and not just have a different controller in feedthrough...